modules/data/tools/smart_recorder/BUILD

```diff
@@ -87,6 +88,18 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "emergency_mode_trigger_beta",
+    srcs = ["emergency_mode_trigger_beta.cc"],
```

Comment:
we already have an emergency_mode trigger, so consider to change the name to logic specific one, for example "hard_break_trigger"